composer.json

```diff
@@ -13,7 +13,7 @@
         "irc": "irc://irc.freenode.net/phergie"
     },
     "require": {
-        "php": "^5.5||^7.0",
+        "php": "^5.6||^7.0",
```

Comment:
It's not a huge deal. I'm fine with this or `"php": ">=5.6.0",`. @WyriHaximus may want to weigh in since he's mostly work on this repo